Sources/Reactive.swift

```diff
@@ -0,0 +1,26 @@
+/// Describes a class which has been extended with reactive elements.
+///
+/// - note: `ExtendedForReactiveness` is only intended for extensions to types
+///         that are not owned by the module. Non-conforming types may carry
+///         first-party reactive elements.
+public protocol ExtendedForReactiveness: class {}
```

Comment:
`ReactiveExtensible`? `ReactiveEnabled`?

> Reactive* somehow sounds like implying non-conforming types of Reactive* are not reactive.

If you think of it as implying they're not `Reactive`, I think it works. 😕